### PR TITLE
186 issues-[instance] remove instance error when wasm file removed 

### PR DESCRIPTION
**Pull request by hunshenshi**
#186

### CODE REVIEW

File: pkg/modules/deploy/deploy.go [hunk 1]
```diff
@@ -239,6 +254,8 @@ func StartInstances(ctx context.Context) error {
 				cmd = enums.DEPLOY_CMD__START
 			case enums.INSTANCE_STATE__STOPPED:
 				cmd = enums.DEPLOY_CMD__STOP
+			case enums.INSTANCE_STATE__LOAD_FAIL:
+				cmd = enums.DEPLOY_CMD__START
```
Comment (saitofun) [inline pkg/modules/deploy/deploy.go:258]: when instance load failed cmd should be `DEPLOY_CMD_UNKNOWN`

File: pkg/modules/resource/resource.go [hunk 2]
```diff
@@ -44,3 +44,12 @@ func FetchOrCreateResource(ctx context.Context, f *multipart.FileHeader) (*model
 	l.Info("get wasm resource from db")
 	return m, err
 }
+
+func CheckResourceExist(ctx context.Context, path string) bool {
```
Comment (saitofun) [inline pkg/modules/resource/resource.go:48]: Remove this method use `IsPathExist` instead

File: pkg/modules/vm/instance_new.go [hunk 3]
```diff
@@ -17,7 +18,7 @@ func NewInstance(ctx context.Context, path string, id types.SFID) error {
 	code, err := os.ReadFile(path)
 	if err != nil {
 		l.Error(err)
-		return err
+		return status.ExtractFileFailed.StatusErr().WithDesc(err.Error())
```
Comment (saitofun) [inline pkg/modules/vm/instance_new.go:21]: Don't need status error

**Review by saitofun — CHANGES_REQUESTED** [review]

File: pkg/modules/deploy/deploy.go (hunk 1 above)
Comment (hunshenshi) [reply pkg/modules/deploy/deploy.go:258]: if cmd still is `DEPLOY_CMD_UNKNOWN`, the instance will can't be started again.
the state `INSTANCE_STATE__LOAD_FAIL` is a error running state,  so the instance should be start again, maybe this time will start right.

File: pkg/modules/resource/resource.go (hunk 2 above)
Comment (hunshenshi) [reply pkg/modules/resource/resource.go:48]: `IsPathExist` is not used to be compatible with other file system.

File: pkg/modules/vm/instance_new.go (hunk 3 above)
Comment (hunshenshi) [reply pkg/modules/vm/instance_new.go:21]: fine

**Review by saitofun — APPROVED** [review]

**Pull request merged**